src/test/java/com/powsybl/openloadflow/sa/AbstractOpenSecurityAnalysisTest.java

```diff
@@ -29,11 +29,7 @@
 import com.powsybl.openloadflow.network.LfBus;
 import com.powsybl.openloadflow.network.NodeBreakerNetworkFactory;
 import com.powsybl.openloadflow.network.SlackBusSelectionMode;
-import com.powsybl.security.LimitViolationFilter;
-import com.powsybl.security.SecurityAnalysisParameters;
-import com.powsybl.security.SecurityAnalysisReport;
-import com.powsybl.security.SecurityAnalysisResult;
-import com.powsybl.security.detectors.DefaultLimitViolationDetector;
+import com.powsybl.security.*;
 import com.powsybl.security.limitreduction.LimitReduction;
```

Comment:
@olperr1 is this on purpose ? (just to be sure - I have no strong opinion about explicit or implicit import - but some projects prefer to always explicit the imports)

Reply:
It wasn't on purpose. I will revert this change.
Thanks.